gumroad/views/admin.php

```diff
@@ -17,10 +17,63 @@
 }
 ?>
 
-<div class="wrap">
-
+<div class="wrap gumroad-wrap">
 	<h2><?php echo esc_html( get_admin_page_title() ); ?></h2>
 
+	<p>The Gumroad plugin makes it easy to use wordpress short codes to add links and embedded
+products from your Gumroad storefront directly into your wordpress pages and posts. There is
+a brand new post and page editor in Version 5.0 of wordpress and onward, Gumroad links and
+embed code will work the same, but only if they are added properly.</p>
+
+	<span style="margin-bottom:65px;display:block;"></span>
+	<h3 class="title"><?php esc_html_e( 'Important SSL Requirement', 'gum' ); ?></h3>
+
+	<p>
```

Comment:
remove everything on here except for the new blocks documentation.

Reply:
Does this mean to remove the old, before 4.9.2 documentation too?